### PR TITLE
Fixes Integer random values should not start with zero

### PR DESCRIPTION
**Pull request by ntkathole**
Issue #5418

### CODE REVIEW

File: tests/foreman/cli/test_classparameters.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def valid_sc_parameters_data():
         },
         {
             u'sc_type': 'integer',
-            u'value': gen_string('numeric', 5),
+            u'value': gen_string('numeric', 5).lstrip('0'),
```
Comment (oshtaier) [inline tests/foreman/cli/test_classparameters.py]: I propose to use `str(gen_integer())` instead

File: tests/foreman/cli/test_classparameters.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def valid_sc_parameters_data():
         },
         {
             u'sc_type': 'integer',
-            u'value': gen_string('numeric', 5),
+            u'value': str(gen_integer()),
```
Comment (san7ket) [inline tests/foreman/cli/test_classparameters.py]: +1

**Comment by san7ket** [comment]
Could you test this with tests.foreman.cli.test_classparameters.SmartClassParametersTestCase.test_positive_update_parameter_type
and get test results

**Comment by ntkathole** [comment]
@oshtaier @san7ket  thanks for review..comment addressed. 

Test results 
```
 pytest tests/foreman/cli/test_classparameters.py::SmartClassParametersTestCase::test_positive_update_parameter_type
========================================================== test session starts ==========================================================
collected 1 item
2017-10-31 16:13:41 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings

tests/foreman/cli/test_classparameters.py .

====================================================== 1 passed in 238.15 seconds ========================================
```
also tried manually generating value  and adding it into code
```
str(gen_integer())
'-8064230584456946206'

data['sc_type'] = 'integer'
data['value'] = '-8064230584456946206'
```

**Pull request merged**